tests/.gitignore

```diff
@@ -12,3 +12,5 @@ dist/
 venv/
 zenko_e2e.egg-info/
 .secrets.env
+.local.env
+.local.make
```

Comment:
did you mean `local.make`?

Reply:
I changed it to keep it inline with the other local config files